src/throttler.guard.ts

```diff
@@ -13,16 +13,24 @@ import { ThrottlerOptions } from './throttler.interface';
 
 @Injectable()
 export class ThrottlerGuard implements CanActivate {
+  storageService: ThrottlerStorage;
+
   constructor(
     @Inject(THROTTLER_OPTIONS) private readonly options: ThrottlerOptions,
-    @Inject(ThrottlerStorage) private readonly storageService: ThrottlerStorage,
+    @Inject(ThrottlerStorage) storageService: ThrottlerStorage,
     private readonly reflector: Reflector,
-  ) {}
+  ) {
+    this.storageService = options.storage || storageService;
```

Comment:
This shouldn't be necessary due to how the `ThrottlerCoreModule` works with setting the `ThrottlerStorage` provider

Reply:
Mhm, couldn't get it to work earlier though.